aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,170 @@
+import { Example } from '../../layouts';
+import { 
+  PageBackGroundExample,
+  PageExample, 
+  PageFullExample,
+  PageNarrowExample,
+  SinglePageContent,
+  PageWideExample 
+} from '../../examples';
+
+Page makes it easy to implement [page layouts](/templates/page-layouts) uniformly across HPE 
+applications. Two components, Page and PageContent, work together to ensure HPE theme-defined 
+padding, horizontal alignment, max-width, and responsive behaviors are applied in a consistent, 
+turn-key manner.
+
+<Example 
+  docs="https://v2.grommet.io/page?theme=hpe"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js"
```

Comment:
Since this example is more visual should we leave out the code? I know in other places we have just the `Preview` open to opinions just a thought

Reply:
Another good question. I was thinking it would be valuable to be able to inspect how that was accomplished, but we provide the ability to do that elsewhere, so not necessarily needed here. I'm leaning towards your instinct to remove. @taysea thoughts?

Reply:
Agree, think we can remove for this example.

Also side note: noticing some funny behavior when I scroll:
<img width="1515" alt="Screen Shot 2022-04-06 at 10 54 53 AM" src="https://user-images.githubusercontent.com/12522275/162037985-0f5bb1bd-c2eb-4fb8-86b4-90a08e182ee8.png">

Reply:
I've fixed that.

Reply:
done